core/src/main/resources/AppConstants.properties

```diff
@@ -372,6 +372,8 @@ message.size.warn.default=30MB
 stub4testtool.configuration=false
 # setting for changing the default timeout in larva
 larva.timeout=10000
+larva.parallel.blacklistDirs=MessageStoreSenderAndListener,QuerySender,Receivers,BlockEnabledSenders,TransactionHandling,BatchFileTransformerPipe,JdbcListener
```

Comment:
Deze moet naar `/test/src/main/resources/DeploymentSpecifics.properties`. Anders staan deze iaf-test scenarios voor alle applicaties uit.

Reply:
Ze staan niet uit, maar ze worden gedraaid in single-thread mode.

(Of dat betekent dat ze dan niet in DeploymentSpecifics horen of wel daarover zeg ik niks; alleen dat ze niet worden geskipt)

Reply:
Slechte verwoording van mij, en mogelijk de property-naam 😄?

Reply:
We kunnen het ook `larva.parallel.skipList` noemen 😉 Maar ik vond blacklist wel passend, omdat ze worden overgeslagen door de parallelle verwerking. Alles wat niet parallel gaat, gaat serieel.

Reply:
Ik zat later vandaag de property as-is verplaatsen. op zich snap ik de naam wel!

Reply:
Top, thanks!

Reply:
Je zou het `larva.parallel.alwaysSingleThreaded` kunnen noemen, of zoiets.